pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -94,6 +95,8 @@ func parseResponse(responses []*es.SearchResponse, targets []*Query, configuredF
 func processLogsResponse(res *es.SearchResponse, target *Query, configuredFields es.ConfiguredFields, queryRes *backend.DataResponse) error {
 	propNames := make(map[string]bool)
 	docs := make([]map[string]interface{}, len(res.Hits.Hits))
+	searchWords := make(map[string]bool)
+	searchWordsRegex := regexp.MustCompile(regexp.QuoteMeta(es.HighlightPreTagsString) + `(.*?)` + regexp.QuoteMeta(es.HighlightPostTagsString))
```

Comment:
nitpick: could this be moved outside of the `processLogsResponse` function? so that we only have to create the regex once, not on every run of function.

Reply:
Fixed in https://github.com/grafana/grafana/pull/63924/commits/80ef4c9455fac1f9c3033f81229dd61fa915eefa. I then added sorting of searchWords string so the order is stable https://github.com/grafana/grafana/pull/63924/commits/fb06a3198d8259901876372f6f8de43207cde501.